application/Espo/Core/Select/Text/Applier.php

```diff
@@ -106,7 +106,7 @@ public function apply(QueryBuilder $queryBuilder, string $filter, FilterParams $
             array_filter(
                 $this->metadataProvider->getTextFilterAttributeList($this->entityType) ?? ['name'],
                 function ($field) use ($fullTextSearchFieldList) {
-                    return !in_array($field, $fullTextSearchFieldList);
+                    return in_array($field, $fullTextSearchFieldList);
```

Comment:
This seems to me wrong.

Reply:
> This seems to me wrong.

You are right. I misunderstood the purpose of `DefautFilter::applyAttribute` method.